### PR TITLE
multi page action bar

### PR DESCRIPTION
**Pull request by yglop**
## About the PR
Action bar now can hold up to 100 items. 10 actions per page.

## Why / Balance
10 slots for action bar is just not enough. This issue become worse with the addition of changeling.

## Technical details
It's just revert of https://github.com/space-wizards/space-station-14/commit/4e84961ef8ca0591a90ca5f30f5c69b557a76318

## Media
Screenshot from 2024-08-07 16-11-59
Screenshot from 2024-08-07 16-12-05
Screenshot from 2024-08-07 16-12-39

## Requirements

- [ ] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by misandrie** [comment]
check if this works on separated ui

**Comment by yglop** [comment]
> check if this works on separated ui
Screenshot from 2024-08-07 17-36-06
Screenshot from 2024-08-07 17-36-12

File: Content.Client/UserInterface/Systems/Actions/ActionUIController.cs [hunk 1]
```diff
@@ -50,14 +51,17 @@ public sealed class ActionUIController : UIController, IOnStateChanged<GameplayS
     [UISystemDependency] private readonly TargetOutlineSystem? _targetOutline = default;
     [UISystemDependency] private readonly SpriteSystem _spriteSystem = default!;
 
+    private const int DefaultPageIndex = 0; // goobstation
```
Comment (whateverusername0) [inline Content.Client/UserInterface/Systems/Actions/ActionUIController.cs:54]: you should prolly add a comment at the start of the class that it's been heavily modified and should not be tampered with (//goobstation)

**Review by whateverusername0 — CHANGES_REQUESTED** [review]
i'm wiznerding you right here right now

**Comment by misandrie** [comment]
> > check if this works on separated ui
> > Screenshot from 2024-08-07 17-36-06
> > Screenshot from 2024-08-07 17-36-12

horrifying

File: Content.Shared/Input/ContentKeyFunctions.cs [hunk 2]
```diff
@@ -90,6 +90,23 @@ public static class ContentKeyFunctions
             {
                 Hotbar1, Hotbar2, Hotbar3, Hotbar4, Hotbar5, Hotbar6, Hotbar7, Hotbar8, Hotbar9, Hotbar0
             };
+        // goobstation
```
[the remark this answers is not in the document]
Comment (yglop) [reply Content.Shared/Input/ContentKeyFunctions.cs:93]: whole file is written that way

[the remark this answers is not in the document]
Comment (whateverusername0) [reply Content.Shared/Input/ContentKeyFunctions.cs:93]: this is so sad, ok resovled

**Comment by JoeHammad1844** [comment]
HOLY SHIT YOU FINALLY FIXED THAT SUPER ANNOYING GLITCH WHERE THE PDA LIGHT ACTION WOULD JUMP POSITIONS WHEN YOU HOLD IT IN YOUR HAND
I WILL NO LONGER WANT TO KILL MYSELF WHEN PLAYING MEDICAL

**Pull request merged**